lib/services/platform-service.ts

```diff
@@ -148,13 +148,20 @@ export class PlatformService implements IPlatformService {
 			platform = platform.toLowerCase();
 
 			var platformData = this.$platformsData.getPlatformData(platform);
+			let appDestinationDirectoryPath = path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME);
+			let lastModifiedTime = this.$fs.exists(appDestinationDirectoryPath).wait() ? 
+				this.$fs.getFsStats(appDestinationDirectoryPath).wait().mtime : null;
 
 			// Copy app folder to native project
+			this.$fs.ensureDirectoryExists(appDestinationDirectoryPath).wait();
 			var appSourceDirectoryPath = path.join(this.$projectData.projectDir, constants.APP_FOLDER_NAME);
-
+			
 			// Delete the destination app in order to prevent EEXIST errors when symlinks are used.
-			this.$fs.deleteDirectory(path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME)).wait();
-			shell.cp("-R", appSourceDirectoryPath, platformData.appDestinationDirectoryPath);
+			_(this.$fs.readDirectory(path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME)).wait())
```

Comment:
you can move `path.join(platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME)` outside of the chain and use the introduced variable in the path.join below